snappy/kernel_os.go

```diff
@@ -222,13 +222,14 @@ func SyncBoot() error {
 		return fmt.Errorf("failed to run SyncBoot: %s", err)
 	}
 
+	overlord := &Overlord{}
```

Comment:
AFAIR we don't want to sprinkle &Overlord{} everywhere. There will be only one instance.